### PR TITLE
Easy: add instructions for testing pytorch/builder

### PR DESCRIPTION
**Pull request by dzhulgakov**
Also makes the repo name configurable for branches in forks

cc @ezyang @seemethere @malfet @svekars @carljparker

### CODE REVIEW

File: .github/scripts/README.md [hunk 1]
```diff
@@ -56,3 +56,9 @@ Exceptions to AWS for self hosted:
 New runner types can be added by committing changes to `.github/scale-config.yml`. Example: https://github.com/pytorch/pytorch/pull/70474
 
 > NOTE: New runner types can only be used once the changes to `.github/scale-config.yml` have made their way into the default branch
+
+### Testing [pytorch/builder](https://github.com/pytorch/builder) changes
+
+In order to test changes to the builder scripts, submit a fake PR against PyTorch that specifies your PR's branch as `builder_branch` in [`.github/templates/common.yml.j2`](https://github.com/pytorch/pytorch/blob/master/.github/templates/common.yml.j2#LL10C9-L10C23). Make sure to regenerate workflow files with `.github/regenerate.sh` (see above).
```
Comment (malfet) [inline .github/scripts/README.md:62]: Hmm, this link would become invalid really soon, should we have an absolute link here (i.e. https://github.com/pytorch/pytorch/blob/26beb46da4b4c778e8162a67711ebe8fcd1408ae/.github/templates/common.yml.j2#LL10C9-L10C23 )
```suggestion
In order to test changes to the builder scripts, submit a fake PR against PyTorch that specifies your PR's branch as `builder_branch` in [`.github/templates/common.yml.j2`](https://github.com/pytorch/pytorch/blob/26beb46da4b4c778e8162a67711ebe8fcd1408ae/.github/templates/common.yml.j2#LL10C9-L10C23). Make sure to regenerate workflow files with `.github/regenerate.sh` (see above).
```

**Review by malfet — CHANGES_REQUESTED** [review]
Thank you for the documentation improvements, but do you mind moving requirements.txt to  https://github.com/pytorch/pytorch/tree/master/.github/requirements  )

**Comment by malfet** [comment]
Also, just curious, would it really be discoverable in pytorch, or would it better to add this doc note to the builder?

**Review by seemethere — APPROVED** [review]
Looks mostly good to me, just hinging on @malfet's comment

**Comment by dzhulgakov** [comment]
I already added it in my other PR to builder/README.md: https://github.com/pytorch/builder/pull/1264/files?file-filters%5B%5D=.md&show-viewed-files=true (given it's just a readme change I'm lazy to separate it out)

**Review by malfet — APPROVED** [review]
LGTM, thank you for the updates.

**Pull request closed without merging**